server/game/cards/06-WoE/EpicPoem.js

```diff
@@ -0,0 +1,22 @@
+const Card = require('../../Card.js');
+class EpicPoem extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            target: {
+                optional: false,
+                cardType: 'creature',
+                controller: 'self',
+                gameAction: ability.actions.sequential([
+                    ability.actions.exalt(),
+                    ability.actions.gainAmber((context) => ({
+                        amount: context.target.amber,
```

Comment:
Can you add a test case for when there are no friendly creatures in play. I belive it might crash.
Better to use then.